libs/deal-ticket/src/deal-ticket.tsx

```diff
@@ -0,0 +1,104 @@
+import { OrderSide, OrderTimeInForce, OrderType } from '@vegaprotocol/wallet';
+import { FormEvent } from 'react';
+import { Order, useOrderState } from './use-order-state';
+import { DealTicketMarket } from './deal-ticket-market';
+import { DealTicketLimit } from './deal-ticket-limit';
+
+const DEFAULT_ORDER: Order = {
+  type: OrderType.Market,
+  side: OrderSide.Buy,
+  size: '1',
+  timeInForce: OrderTimeInForce.IOC,
+};
+
+// TODO: Consider using a generated type when we have a better solution for
+// sharing the types from GQL
+export interface Market {
+  id: string;
+  decimalPlaces: number;
+  tradableInstrument: {
+    instrument: {
+      product: {
+        quoteName: string;
+        settlementAsset: {
+          id: string;
+          symbol: string;
+          name: string;
+        };
+      };
+    };
+  };
+  tradingMode:
+    | 'BatchAuction'
+    | 'Continuous'
+    | 'MonitoringAuction'
+    | 'OpeningAuction';
+  state:
+    | 'Active'
```

Comment:
can we not use an enum or type for this?

Reply:
The problem is the enums are generated in apps/trading and shouldn't be imported into the the deal ticket package. We need them in a shared package

Reply:
I've gone ahead and added a types lib